tests/pr_test.go

```diff
@@ -2,10 +2,13 @@
 package test
 
 import (
+	"log"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"gopkg.in/yaml.v2"
```

Comment:
I think this can be changed to `"gopkg.in/yaml.v3"`

Reply:
It was because I used a lowercase instead of uppercase in the config struct :(